cfy_manager/components/restservice/scripts/create_amqp_resources.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+#########
+# Copyright (c) 2016 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+import sys
+import json
+
+from manager_rest.amqp_manager import AMQPManager
+from manager_rest.constants import DEFAULT_TENANT_ID
+from manager_rest.storage import models, get_storage_manager
+
+
+def _get_amqp_manager(config):
+    return AMQPManager(
+        host=config['amqp_host'],
+        username=config['amqp_username'],
+        password=config['amqp_password']
+    )
+
+
+def _get_default_tenant():
+    sm = get_storage_manager()
+    return sm.get(models.Tenant, DEFAULT_TENANT_ID)
+
+
+if __name__ == '__main__':
+    # We're expecting to receive as an argument the path to the config file
+    assert len(sys.argv) == 2, 'No config file path was provided'
```

Comment:
i know this is a copy from the previous version, but assert is a weird thing to use for this; should be a raise (possibly ValueError), or better yet argparse

Reply:
You're right in theory about argparse. However, this really shouldn't be a script that users will ever run, so I don't want to give the appearance of one. I will raise an error though